packages/react-native-lockdown/README.md

```diff
@@ -0,0 +1,114 @@
+# LavaMoat React Native lockdown
+
+LavaMoat React Native lockdown sets up [Hardened JavaScript][hardened-js-ext] in React Native apps.
+
+This is done by repairing Hermes (or JSC) intrinsics at Metro's serializer option _getPolyfills_:
+
+> An optional list of polyfills to include in the bundle. The list defaults to a set of common polyfills for Number, String, Array, Object...
+
+with appropriate [lockdown options][lockdown-options] for React Native, then hardening the intrinsics at Metro's first serializer option _getRunModuleStatement_:
+
+> Specify the format of the initial require statements that are appended at the end of the bundle. By default is `\_\_r(${moduleId});`.
+
+without patching React Native [Core][init-core-ext] where global variables typical in most JavaScript environments are set up.
+
+## Install
+
+```shell
+npm i @lavamoat/react-native-lockdown
+```
+
+or
+
+```shell
+yarn add @lavamoat/react-native-lockdown
+```
+
+> [!NOTE]
+> @react-native/js-polyfills is a peer dependency, likely stemming from your React Native version (example):
+>
+> ```log
+> project@0.0.1
+> ├─┬ @react-native/metro-config@0.72.12 (devDependencies)
+> │ └── @react-native/js-polyfills@0.72.1
+> └─┬ react-native@0.72.15 (dependencies)
+>   └── @react-native/js-polyfills@0.72.1 deduped
+> ```
+
+## Usage
+
+### Babel config
+
+```js
+// babel.config.js
+module.exports = {
+  ignore: [/\/ses\.cjs$/, /\/ses-hermes\.cjs$/],
+  presets: ['module:metro-react-native-babel-preset'],
```

Comment:
update to latest
- https://github.com/facebook/react-native/blob/v0.72.17/packages/react-native/template/babel.config.js
  - module:metro-react-native-babel-preset
- https://github.com/react-native-community/template/blob/main/template/babel.config.js
  - module:@react-native/babel-preset

Reply:
include note for old [unsupported](https://github.com/reactwg/react-native-releases/blob/main/docs/support.md) <=0.72.x users